.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,39 @@
+name: Benchmarks
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  bench:
+    name: Run benchmarks daily
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout Sources
+        uses: actions/checkout@v2.4.0
+
+      - name: Install Rust nightly toolchain
+        uses: actions-rs/toolchain@v1.0.7
+        with:
+          profile: minimal
+          toolchain: nightly
+          override: true
+
+      - name:                      Rust Cache
```

Comment:
Is this large space just a github formatting?

Reply:
spaces but I copy-pasted this from an old commit, fixed now